indico/web/client/js/react/components/index.js

```diff
@@ -5,6 +5,14 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
+export {
```

Comment:
Why was this move needed? (Requiring a particular export order seems rather fragile!)

Reply:
I forgot to comment this in one of the commits. The cause is this exact webpack-contrib/mini-css-extract-plugin#382
A clearer explanation is also [here](https://github.com/facebook/create-react-app/issues/5372#issuecomment-685932009).

It's possibly safe to ignore these warnings, however, this was the only change required to silence then alll.